changelog.md

```diff
@@ -18,8 +18,8 @@
 * The setting for simulatorInjectLineNoise has changed from `None` to `none`
 * connect() will now fail if already connected
 * The constructor will throw an error now if an invalid option is passed
-* The undocumented `.connected` field has been removed, replaced by `.isConnected()`
-* The undocumented `.streaming` field has been removed, replaced by `.isStreaming()`
+* The undocumented `.connected` field has been removed, replaced by `.isConnected()`. Removed from docs.
+* The undocumented `.streaming` field has been removed, replaced by `.isStreaming()`. Removed from docs.
```

Comment:
oh my goodness I totally never saw that these were actually documented, I would never have changed them